src/tests/simulation/tests/integration_tests/scale/schedulability.rs

```diff
@@ -0,0 +1,558 @@
+// Copyright 2023 RisingWave Labs
```

Comment:
I agree. @shanicky pointed out to me that I should move the testing logic. The suggestion was that this should be done as a follow-up PR. Would that be acceptable to you? I would like to get this functionality merged soon, since we have people waiting for it.